PatientMonitoringSystem/PatientMonitoringSystem/Enums/ModuleTypeEnum.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PatientMonitoringSystem.Enums
+{
+	public enum ModuleType
```

Comment:
@harrytpstudent No need to call the class "ModuleTypeEnum". Just "ModuleType" would do.

Reply:
Sure.  Will amend now.